RELEASE/scripts/autoscend/paths/actually_ed_the_undying.ash

```diff
@@ -1443,7 +1443,7 @@ boolean LM_edTheUndying()
 		return true;
 	}
 	// L8 quest is all 1 Ka zones for Ed (unlikely to survive Ninja Snowmen Assassins so they don't count)
-	if (L8_trapperStart() || L8_trapperGround() || L8_trapperGroar())
+	if (L8_trapperGround() || L8_trapperExtreme() || L8_trapperGroar())
 	{
 		return true;
 	}
```

Comment:
* L8_trapperStart() is no longer necessary. starting as part of L8_trapperGround()
* ed was relying on backtracking code in L8_trapperGroar() which went and executed L8_trapperExtreme(). since this backtracking code has been removed I added doing L8_trapperExtreme() first.

Also I am not sure if we should. but the above can be replaced instead with:
```suggestion
	set_property("auto_L8_ninjaSkip", true);		//give up on ninja route for this ascension
	if(L8_trapperQuest())
	{
		return true;
	}
```
```


```

Reply:
Or if there is a scenario where we do want to do ninjas as ed. I can add code for that instead

Reply:
Oh wait. there is. softcore ed wants to pull the ninja gear.